pkg/synchronizer/synchronizer_test.go

```diff
@@ -74,6 +74,16 @@ func TestToSecretDataWithEnv(t *testing.T) {
 	}, payload)
 }
 
+func TestToEnvMultilineEncoded(t *testing.T) {
+	envPayload       = []byte("FOO=\"one line\\ntwo line\\n\"\n")
```

Comment:
I'd like to see the addition of a second environment variable (e.g. `BAR=BAZ`) in the test payload as well.

Reply:
Will do